.circleci/config.yml

```diff
@@ -270,7 +270,7 @@ jobs:
             cf conduit tariff-"<< parameters.service >>"-staging-postgres -- psql $PGDATABASE < tariff-"<< parameters.service >>"-production-postgres.psql
             cf conduit tariff-"<< parameters.service >>"-staging-postgres -- psql $PGDATABASE < db/seeds/staging.sql
       - run_migrations:
-          service: staging
```

Comment:
We should only be running staging migrations? What am I missing?

Reply:
Yeh - that was the mistake I made - bit confusing, service is XI or UK